### PR TITLE
支持中文路径

### PR DESCRIPTION
**Pull request by sunlingzhang**

### CODE REVIEW

**Comment by GreatV** [comment]
please install pre-commit and run, to fix code style.

File: PPOCRLabel.py [hunk 1]
```diff
@@ -1248,7 +1248,7 @@ def createPolygon(self):
 
     def rotateImg(self, filename, k, _value):
         self.actions.rotateRight.setEnabled(_value)
-        pix = cv2.imread(filename)
+        pix = cv2.imdecode(np.fromfile(filename, dtype=np.uint8), cv2.IMREAD_COLOR)
         pix = np.rot90(pix, k)
         cv2.imwrite(filename, pix)
```
Comment (GreatV) [inline PPOCRLabel.py:1253]: Is `cv2.imwrite` affected by Chinese paths?

Comment (sunlingzhang) [reply PPOCRLabel.py:1253]: Yes, for the Chinese path,  `cv2.imwrite(file_name, img)` should be modified to  `cv2.imencode(".jpg", img).tofile(file_name)`.

Comment (Gmgge) [reply PPOCRLabel.py]: 根据linux（centos7.9），windows系统（win10）测试，以及opencv官方[issue](https://github.com/opencv/opencv/issues/8512)确认，在windows下，imread、imwrite不兼容中文路径。

**Review by GreatV — APPROVED** [review]
LGTM

**Pull request merged**